test/tasks/native_extensions_test.rb

```
@@ -216,3 +216,137 @@ def test_build_native_gem_exports_exts_and_exts_dir
+        DLLIB = $(TARGET).bundle
+        target_prefix = /bigdecimal
+      MAKEFILE
+
+      mock_task(Kompo::WorkDir, path: work_dir)
+      mock_task(Kompo::FindNativeExtensions, extensions: [
+        {
+          dir_name: ext_dir,
+          gem_ext_name: "bigdecimal",
+          is_rust: false,
+          is_prebuilt: true
+        }
```

RuboCop (1.82.1)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)